lighttpd/datadog_checks/lighttpd/lighttpd.py

```diff
@@ -66,9 +66,10 @@ def __init__(self, name, init_config, instances):
         self.assumed_url = {}
 
         if 'auth_type' in self.instance:
```

Comment:
That condition doesn't seem required anymore, the second line takes care of that.